labs/lab-4-jquery/restaraunt.js

```diff
@@ -0,0 +1,7 @@
+class Restaurant {
```

Comment:
You're also not using this. I'd have put your table-making code in here:

```
for(i = 1; i < width*height; i++) {   
  var newTable = new Table(i, "available");
  this.tables.push(newTable);
  newTable.render();
}
```

...but as it is, you're not using it, so get rid of the whole file!